src/router/routes.js

```diff
@@ -1,16 +1,21 @@
+import { modelizerRoute, modelizerShortRoute } from 'src/router/routes/modelizerRoutes';
```

Comment:
All route have to have the same Name of their file's name. So you have to capitalize them.